src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerfileFunctionalTest.groovy

```diff
@@ -1,13 +1,24 @@
 package com.bmuschko.gradle.docker.tasks.image
 
 import com.bmuschko.gradle.docker.AbstractFunctionalTest
-import org.apache.commons.lang.exception.ExceptionUtils
-import org.gradle.api.tasks.TaskExecutionException
 import org.gradle.testkit.runner.BuildResult
 
+import java.nio.file.Files
+import java.nio.file.Paths
+
 class DockerfileFunctionalTest extends AbstractFunctionalTest {
     static final String DOCKERFILE_TASK_NAME = 'dockerfile'
 
+
+    private void setupDockerTemplateFile() {
+        File source = new File(TestConfiguration.class.getClassLoader().getResource("Dockerfile.template").toURI())
+        if (source.exists()) {
+            File resourcesDir = new File(projectDir, 'src/main/docker/')
+            resourcesDir.mkdirs()
```

Comment:
Lets do a check to ensure this succeeds or is true. It's a long shot but we don't want to silently ignore failures.